ROADMAP.md

```diff
@@ -0,0 +1,226 @@
+the [core repo][wg-ctc]. We wish to bring the [guide documentation][guide-docs]
+into the core repo.
+
+The existing [node doctool][ref-doctool] is specifically built to _only_
+generate API docs. Until we can build the guide docs with Node's `make doc`
+command, guide documentation will be copied from the website repo into the core
+repo, but not *removed* from the website repo.
+
+As a result, we are investigating using [remark][ref-remark] to build the
+docs. The stages of this project are as follows:
+
+1. Use remark to build just the guides alongside the existing doctool.
+2. Remove the guides from the website repository.
+3. Identify and install the necessary remark plugins to faithfully render the
+   API documentation.
+4. Generate all documentation using remark. Fix lint issues pointed out by
+   `remark-lint`.
+5. Once no linting issues remain, wire up docs linting as part of Node's 
+   `make test`.
+
+Once this project is complete, all doc style rules will be handled by remark,
+which includes line lengths, code samples (via eslint), markdown bullet and
+emphasis styles, and link checking. We will have a solid basis on which to add
+spelling and grammar checkers in the future, making it easier to maintain the
+docs going forward.
+
+##### Subtasks
+
+Want to pitch in? Look here! If someone's already working on the task, see
+if they need help.
+
+* Identifying Remark plugins — **@qard**, others! [nodejs/docs#61](https://github.com/nodejs/docs/issues/61)
+* Initial Remark integration — **@qard** [nodejs/node#4866](https://github.com/nodejs/node/pull/4866)
+* Codify linting rules
+  * Apply linting rules across API docs
+* HTML layout for guide docs
+* HTML layout update for API docs to include refs to guide docs
+
+#### :two: Guide and Topic Docs
+
+A _very_ close second to the work on the documentation tooling is the work on
+identifying and creating new guides and topic documentation. These two terms
+come up a lot, to clarify what they mean, here's an except from our [getting
+started][ref-getting-started] guide:
+
+> 1. **Guide** documents explain processes to help the reader learn a concept
+> in service of their larger goal. Usually a guide has the reader build
+> something — a little webserver, or CLI — and explains the concept it's trying
+> to convey using examples from the readers experience with that code. Guides
+> are great for introducing new concepts in a comfortable way, by letting the
+> reader "simulate" the process of what development will be like using those
+> concepts.
+> 2. **Topic** documents explain concepts to help the reader make a decision.
+> They are a great place for "deep dive" information, and to handle anything
+> that's fairly intricate.
+> 3. **Reference** documents explain capabilities to help the reader achieve
+> their goal. API documentation is the primary example of this.
+
+Node currently has **reference** documentation, but because of the lack of
```

Comment:
Node.js

On a separate note, we tend to switch between "Node.js, Node and Node's" in this document. We should really use "Node.js and Node.js'" in all instances.